src/aleph/web/controllers/prices.py

```diff
@@ -81,11 +81,13 @@ async def message_price(request: web.Request):
         message = await get_executable_message(session, request.match_info["item_hash"])
 
         content: ExecutableContent = message.parsed_content
-
-        if content.payment and content.payment.is_stream:
-            required_tokens = compute_flow_cost(session=session, content=content)
-        else:
-            required_tokens = compute_cost(session=session, content=content)
+        try:
+            if content.payment and content.payment.is_stream:
+                required_tokens = compute_flow_cost(session=session, content=content)
+            else:
+                required_tokens = compute_cost(session=session, content=content)
+        except RuntimeError as e:
```

Comment:
Can you explain why is a `RuntimeError` error expected here ? Is that error not too generic ?

Reply:
[Where it can failed](https://github.com/aleph-im/pyaleph/blob/09b6d5b5ed50a625dcaf82fe3eaaef7f48a4cf63/src/aleph/services/cost.py#L70)
It's in order to handle this error if volume is removed but not the program